api/resolvers/wallet.js

```diff
@@ -779,6 +784,7 @@ async function upsertWallet (
   if (testCreateInvoice) {
     try {
       await testCreateInvoice(data)
+      await validate({ data, settings, skipGenerated: false })
```

Comment:
The changes in this file are not related to this pr specifically.
See https://github.com/stackernews/stacker.news/pull/1769

Reply:
So this means this PR is based on #1769?

Reply:
yes